### PR TITLE
Show only online users

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: client/app/scripts/superdesk-users/users.js [hunk 1]
```diff
@@ -252,13 +253,26 @@
                 };
 
             if (params.q) {
-                criteria.where = JSON.stringify({
+                var query = {
```
Comment (akintolga) [inline client/app/scripts/superdesk-users/users.js]: There is a UserListService in the same file on line 90. Maybe we should use that service here. I have another PR (https://github.com/superdesk/superdesk/pull/404) where I supposed to do the same actually. What do you think?

**Comment by amagdas** [comment]
:+1:

File: client/app/scripts/superdesk-users/users.js (hunk 1 above)
Comment (ioanpocol) [inline client/app/scripts/superdesk-users/users.js]: I will merge your PR and then I will make the refactoring.

**Comment by petrjasek** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by ioanpocol** [comment]
@akintolga I didn't refactored yet the users to use the cached service. I think that the effort to do this is not so small as I thought initially, so I will add a new PR for it.

**Comment by akintolga** [comment]
Fair enough. I am also more than happy to look at the re-factoring that bit in the next few days as I supposed to do that in the first place.

**Comment by ioanpocol** [comment]
OK

**Pull request merged**